src/Compilers/Core/Portable/Symbols/Attributes/UnmanagedCallersOnlyAttributeData.cs

```diff
@@ -16,6 +16,7 @@ internal sealed class UnmanagedCallersOnlyAttributeData
         internal static readonly UnmanagedCallersOnlyAttributeData Uninitialized = new UnmanagedCallersOnlyAttributeData(callingConventionTypes: ImmutableHashSet<INamedTypeSymbolInternal>.Empty, isValid: false);
         internal static readonly UnmanagedCallersOnlyAttributeData AttributePresentDataNotBound = new UnmanagedCallersOnlyAttributeData(callingConventionTypes: ImmutableHashSet<INamedTypeSymbolInternal>.Empty, isValid: false);
         private static readonly UnmanagedCallersOnlyAttributeData PlatformDefault = new UnmanagedCallersOnlyAttributeData(callingConventionTypes: ImmutableHashSet<INamedTypeSymbolInternal>.Empty, isValid: true);
+        internal static readonly UnmanagedCallersOnlyAttributeData Invalid = new UnmanagedCallersOnlyAttributeData(callingConventionTypes: ImmutableHashSet<INamedTypeSymbolInternal>.Empty, isValid: false);
```

Comment:
>Invalid [](start = 67, length = 7)

What advantage do we get from using this instance?

Reply:
`null` is a valid case of `UnmanagedCallersOnlyAttributeData`, which means that the method has the default calling convention. I felt that the error experience would be better if we treated a mangled `UnmanagedCallersOnly` application as default unmanaged, rather than default managed.

---
In reply to: [489025845](https://github.com/dotnet/roslyn/pull/47651#discussion_r489025845) [](ancestors = 489025845)